src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -210,12 +211,29 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
    * @return
    */
   def getDepositState(user: String, id: UUID): Try[StateInfo] = {
+    trace(user, id)
+    getDepositStateManager(user, id)
+      .flatMap(_.getStateInfo)
+  }
+
+  def forceChangeState(user: String, id: UUID, newStateInfo: StateInfo, update: Boolean): Try[String] = {
+    trace(user, id)
+    for {
+      stateManager <- getDepositStateManager(user, id)
+      oldStateInfo <- stateManager.getStateInfo
+      _ = if (update) stateManager.saveInDraft(newStateInfo)
+    } yield
+      s"""OLD: ${ toJson(oldStateInfo) }
+         |NEW: ${ toJson(newStateInfo) }
+         |""".stripMargin
```

Comment:
Not sure if JSON is the best format to be used here.

Reply:
Then please specify what you want. See the unit tests.